teuthology/provision/pelagos.py

```diff
@@ -71,18 +71,22 @@ def __init__(self, name, os_type, os_version=""):
             self.os_name = os_type
         self.log = log.getChild(self.name)
 
-    def create(self):
+    def create(self, init_only=False):
         """
         Initiate deployment via REST requests and wait until completion
-
+        :param warn: if set, function exits just after initiation, down
```

Comment:
Hm.... did you mean `init_only` here instead of `warn`?

Reply:
Correct assumption. Fixed.